sos/report/plugins/ceph_osd.py

```diff
@@ -36,6 +36,39 @@ class CephOSD(Plugin, RedHatPlugin, UbuntuPlugin):
 
     def setup(self):
         microceph_pkg = self.policy.package_manager.pkg_by_name('microceph')
+        cmds = [
+            # will work pre quincy
+            "dump_reservations",
+            "bluestore bluefs available",
+            # will work quincy onward
+            "bluestore bluefs device info",
+            "bluefs stats",
+            "config diff",
+            "config show",
+            "dump_blocklist",
+            "dump_blocked_ops",
+            "dump_historic_ops_by_duration",
+            "dump_historic_slow_ops",
+            "dump_mempools",
+            "dump_ops_in_flight",
+            "dump_op_pq_state",
+            "dump_osd_network",
+            "dump_pgstate_history",
+            "dump_recovery_reservations",
+            "dump_scrubs",
+            "get_mapped_pools",
+            "dump_watchers",
+            "log dump",
+            "list_devices",
+            "list_unfound",
+            "perf dump",
+            "perf histogram dump",
+            "objecter_requests",
+            "ops",
+            "status",
+            "version",
+        ]
```

Comment:
Appreciate the alphabetical ordering, to prevent duplicates like you detected :)